src/resources/Resource.js

```diff
@@ -11,12 +11,20 @@ export default class Resource {
     return url
   }
 
-  async request ({ path, fields = [], params = {}, method = 'get' }) {
+  async request ({
+    path,
+    fields = [],
+    params = {},
+    method = 'get',
```

Comment:
I know you didn't do this, but can you change method to uppercase `GET` to standardise please?